views/006_config_views.sql

```diff
@@ -168,6 +169,11 @@ CREATE OR REPLACE VIEW config_tags AS
   SELECT d.key, d.value
   FROM configs JOIN json_each_text(tags::json) d ON true GROUP BY d.key, d.value ORDER BY key, value;
 
+-- config_labels
```

Comment:
we probably need a config_tags as well

Reply:
the previous `config_tags` view also exists.